src/onegov/form/display.py

```diff
@@ -160,11 +152,11 @@ def __call__(self, field: 'Field', **kwargs: Any) -> Markup:
             url = self.ensure_vimeo_embedded_url(data)
 
         if url:
-            return Markup(self.video_page.render(url=self.escape(url)))
+            return self.video_template.format(url=self.escape(url))
 
         # for other sources we try to render video url but also provide
         # the link
```

Comment:
I think this is incredibly ill-advised, since directories can accept submissions from the public it would be quite easy to inject harmful pages into iframes that would get loaded immediately without a user having to first click a link. It's already kind of debatable whether embedding youtube/vimeo videos is a good idea unless we can sandbox the iframe to a degree where it can't track where it was embedded from.

Reply:
I will try to sandbox the iframe.
Regarding 'other' non-youtube, non-vimeo sources, would you not render the video at all, just show the link?

Reply:
just show the link in that case, yes